syntax/syntax.go

```diff
@@ -9,6 +9,47 @@ package syntax
 type Node interface {
 	// Span returns the start and end position of the expression.
 	Span() (start, end Position)
+
+	// Comments returns the comments associated with this node.
+	// It returns nil if RetainComments was not specified during parsing.
```

Comment:
"during parsing, or if AllocComments was not called."

Reply:
Done